asf_search/search/error_reporting.py

```diff
@@ -19,7 +20,7 @@ def report_search_error(search_options: ASFSearchOptions, message: str):
     message=f"Error Message: {str(message)}\nUser Agent: {user_agent} \
     \nSearch Options: {{\n{search_options_list}\n}}"
 
-    response = requests.post(f'https://{ERROR_REPORTING_ENDPOINT}', data={'Message': "This error message and info was automatically generated:\n\n" + message})
+    response = requests.post(f'https://{INTERNAL.ERROR_REPORTING_ENDPOINT}', data={'Message': "This error message and info was automatically generated:\n\n" + message})
```

Comment:
In light of the removal of the constants name-spacing that tried to sneak in, is this change actually intended as well?